packages/genesys-spark-components/src/components/stable/gux-toast/example.html

```diff
@@ -19,7 +21,9 @@ <h2>Gux Success Toast</h2>
 
 <gux-toast toast-type="success" aria-live="polite">
   <div slot="message">Success toast with Link without Title</div>
-  <a slot="link" href="#">Link</a>
+  <a slot="link" href="#"
+    >Link<gux-icon icon-name="external-link" decorative></gux-icon
+  ></a>
 </gux-toast>
 
 <h2>Gux Warning Toast</h2>
```

Comment:
There is one toast in here that has a Link without and icon

Reply:
oops. that's a miss on my part

Reply:
I think this just needs to be updated I did a re-review all looks good will approve after this 👍

Reply:
Updated